swingtix/bookkeeper/test_model.py

```diff
@@ -352,7 +350,6 @@ def test_book_set_basic(self):
             description="Expenses",
             positive_credit=False
         )
-        expense.save()
```

Comment:
Hi Omar!  Thanks for your contributions to bookkeeper!

I don't understand why .save() needs to be removed here.. I must be missing something?